dbcon/joblist/groupconcat.cpp

```diff
@@ -500,8 +476,8 @@ void GroupConcator::outputRow(std::ostringstream& oss, const rowgroup::Row& row)
                 }
                 else
                 {
-                    long double dblVal = uintVal / pow(10.0, (double)scale);
-                    oss << fixed << setprecision(scale) << dblVal;
+                    oss << fixed << datatypes::Decimal(datatypes::TSInt64((int64_t) uintVal),
+                                                       scale, row.getPrecision(*i));
```

Comment:
I think casting an `uint64_t` to a `int64_t` here can overflow, we should use the `TSInt128` version here.

Reply:
It should not overflow. scale is not zero. So the number is at least 10 times smaller. It fits into int64_t.
I think there should be a comment about this.